src/main/java/duke/iomethods/IOMethods.java

```diff
@@ -0,0 +1,97 @@
+package duke.iomethods;
+
+import duke.exception.DukeException;
+
+import java.util.ArrayList;
+
+public class IOMethods {
+    public static ArrayList<String> splitToTwo(String line, String delimiter) {
+        ArrayList<String> words = new ArrayList<>();
+        int divider = line.indexOf(delimiter);
+
+        words.add(line);
+
+        if (divider != -1) {
+            words.set(0, line.substring(0, divider));
+            words.add(line.substring(divider+1));
+        }
+        return words;
+    }
+
+    public static void printWithDivider(String stringWithinDivider) {
+        String breakLine = "\t____________________________________________________________";
+        System.out.println(breakLine);
+        stringWithinDivider = stringWithinDivider.replace("\n", "\n\t");
+        System.out.println("\t" + stringWithinDivider);
+        System.out.println(breakLine);
+    }
+
+    public static void errorHandler(String input) throws DukeException {
```

Comment:
This is method is [quite long (>30 LoC)](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods). Consider applying [SLAP](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard) for each of the commands for further abstraction.